nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/CoreSession.java

```diff
@@ -1268,7 +1268,7 @@ DocumentModel restoreToVersion(DocumentRef docRef, DocumentRef versionRef, boole
      *
      * @param docRef the document reference
      * @return the lock info that was set
-     * @throws LockException if the document is already locked
+     * @throws LockException if the document is already locked by another user.
```

Comment:
No final `.` on the various `@param` `@return` and `@throws` texte, as it's not a full sentence.
See https://www.oracle.com/technetwork/java/javase/tech/index-137868.html#@param
> End the phrase with a period only if another phrase or sentence follows it.